packages/core/src/internal/io.js

```diff
@@ -71,46 +71,51 @@ export function race(effects) {
   return makeEffect(effectTypes.RACE, effects)
 }
 
-function getFnCallDesc(meth, fn, args) {
+const callFnEffects = ['call', 'apply', 'cps', 'fork', 'spawn']
+
+function getFnCallDescriptor(fnDescriptor, args) {
   if (process.env.NODE_ENV !== 'production') {
-    check(fn, is.notUndef, `${meth}: argument fn is undefined`)
+    check(fnDescriptor, is.notUndef, `${callFnEffects.join('/')}: argument fn is undefined`)
   }
 
   let context = null
-  if (is.array(fn)) {
-    ;[context, fn] = fn
-  } else if (fn.fn) {
-    ;({ context, fn } = fn)
```

Comment:
Are we going to drop the support for `{ context, fn }`?

Reply:
Oh, this is a bummer 😅 had no idea this is publicly supported, gonna have to restore it & write a test for this